### DISCUSSION TITLE
calendar days aren't filled in

### DISCUSSION THREAD

**Discussion by eastblue056** [iSoron/uhabits#2002] [Help & FAQ]
![Screenshot_20240528_231321_Habits](https://github.com/iSoron/uhabits/assets/8240372/059d8300-09bb-42d6-813a-783fca3ed9c3)

I set this habit two 60 mins per week in they beginning & change it to 120 mins per week a while a go. somehow the calender days won't aren't colored in, even though I successfully checked my habit on these days.

I hope someone can help me fix this issue.

Thanks in advance

**Comment by RetriedAgain** [comment]
I have tried sth. along these lines myself by increasing a "min. 5 minutes per day"-habit to "min. 10 minutes per day". For all past events, "10 minutes per day" were then required and all boxes became unchecked.

I believe it is basically a code issue, not an issue just on your end.

If one were to try and solve it within changing code of the app, maybe one could save all the days where one changed the habit with their value, e.g. in your case one would have to technically save the new value, 120, if it is a minimum/maximum as well as the date when you changed it.
